.github/workflows/maven-snapshot.yml

```diff
@@ -10,18 +10,20 @@
 # governing permissions and limitations under the License.
 #
 
-name: Publish package to the Maven Central Snapshot Repository
-on: workflow_dispatch
-
+name: Publish Snapshot
+on:
+  push:
```

Comment:
We updated to `workflow_dispatch:` [here](https://github.com/adobe/aepsdk-userprofile-android/blob/dev-v3.0.0/.github/workflows/maven-snapshot.yml#L3).

So won't automatically running on push to staging branch.

Reply:
👍🏼  Sounds good